Multisig/UI/Assets/Collectibles/CollectibleViewModel.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CollectiblesViewModel.swift
+//  Multisig
+//
+//  Created by Moaaz on 7/8/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct CollectibleViewModel: Identifiable {
+    let id: UUID
+    let name: String
+    let description: String
+    let imageURL: URL?
+    let hasName: Bool
+    
+    init() {
+        id = UUID()
+        name = ""
+        description = ""
+        imageURL = nil
+        hasName = false
+    }
+
+    init(collectible: Collectible) {
+        id = UUID()
+
+        name = collectible.name ?? "Unknown"
+        hasName = !(collectible.name == nil || collectible.name!.isEmpty)
```

Comment:
is it the same as (name in ["", "Unknown"])?
I just wonder if we need this additional convenience property. And I would rather make all convenience properties as calculated